docs/surveys/salary-survey/2022.rst

```diff
@@ -14,7 +14,2069 @@ Write the Docs Salary Survey 2021 Results
+   +------------------------+------+---------------+-----+
+   | Japanese Yen           | JPY  | 0.007657      | 1   |
+   +------------------------+------+---------------+-----+
+   | South Korean Wan       | KRW  | 0.0008        | 1   |
+   +------------------------+------+---------------+-----+
+   | New Taiwan Dollar      | TWD  | 0.0333        | 1   |
+   +------------------------+------+---------------+-----+
+   | Swiss Franc            | CHF  | 1.048015      | 1   |
+   +------------------------+------+---------------+-----+
+   | Serbian Dinar          | RSD  | 0.0089        | 1   |
+   +------------------------+------+---------------+-----+
+   | Icelandic Krona        | ISK  | 0.0073        | 1   |
+   +------------------------+------+---------------+-----+
+   | Singapore Dollar       | SGD  | 0.725526      | 1   |
+   +------------------------+------+---------------+-----+
+   | Kenyan Shilling        | KES  | 0.008         | 1   |
+   +------------------------+------+---------------+-----+
+
+Salary
+------
+
+.. container:: question
+   
+   .. rubric:: What we asked
+
+   What currency are you paid in?
+
+   Would you prefer to enter your salary as a yearly or monthly amount?
+   
+   What is your salary (including tax)?
+
+Employees were given the option of entering their salary as a monthly or as a yearly figure, to cater for different countries where one or the other is the norm. Monthly salaries were multipled by 12 in order to compare them to annual salaries. 
+
+In 2022, 79% of employee respondents entered an annual salary figure, and 21% entered a monthly figure. 
+
+As 95.2% of respondents reported working full-time hours (more than 30 per week), the salaries for those working part-time hours (less than 30 per week) have been omitted from the figures in this section. 
+
+Median salary
+~~~~~~~~~~~~~
+
+The median employee salary across all regions was USD $79,506 (meaning half of all respondent earned more, and half earned less). This is lower than the overall median in 2021 and 2022 (USD $80,870 and USD $80,000 respectively) but higher than the overall median in 2019 (USD $74,500).
+
+Median salary by respondent region
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Given the range of socio-economic differences in the countries in the survey results, median salary figures broken down by country of residence of employee is more useful than overall median salary.
+
+In order to protect the privacy of respondents, median salaries are not shown for any country or region with less than 10 respondents. Countries excluded by this condition are:
+
+- Armenia
+- Belarus
+- Belgium
+- Brazil
+- Croatia
+- Czech Republic
+- Estonia
+- Finland
+- France
+- Greece
+- Hungary
+- Iceland
+- Ireland
+- Italy
+- Japan
+- Kazakhstan
+- Netherands
+- New Zealand
+- Norway
+- Poland
+- Portugal
+- Serbia
+- Singapore
+- Slovenia
+- South Africa
+- South Korea
+- Spain
+- Sweden
+- Switzerland
+- Taiwan
+- Ukraine
```

Comment:
If these sample sizes are so small, is there any danger that listing the countries actually deanonimizes the result?

Reply:
I don't think so, because none of the salary figures or demographic data is linked to any individual country (or region in the case of regions with only a handful of responses). I put this in there as a kind of shout out/acknowledgement of all the respondents from countries and regions that aren't otherwise mentioned in the report.